### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SRGOM**
Added link to "awesome-powershell“

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,10 @@ There is also the community driven PowerShell Slack Team which you can sign up f
 
 [Slack Sign up]: http://slack.poshcode.org
 
+## Addons and libraries
```
Comment (iSazonov) [inline README.md]: I believe "Add-Ons" is better term.
Also please add it in https://github.com/PowerShell/PowerShell/blob/master/.spelling

**Review by iSazonov — COMMENTED** [review]
Leave a comment

**Comment by iSazonov** [comment]
Please remove trailing spaces:
> C:\projects\powershell\README.md: 84: MD009 Trailing spaces

File: README.md [hunk 2]
```diff
@@ -79,6 +79,10 @@ There is also the community driven PowerShell Slack Team which you can sign up f
 
 [Slack Sign up]: http://slack.poshcode.org
 
+## Add-ons and libraries
+
+[Awesome powershell](https://github.com/janikvonrotz/awesome-powershell) is a great curated list of add-ons and resources.
```
Comment (SteveL-MSFT) [inline README.md]: Correct casing PowerShell

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Review by SRGOM — COMMENTED** [review]
My approve option is greyed out sorry...

**Comment by iSazonov** [comment]
@SRGOM Approve option is for reviewers only.

File: README.md (hunk 2 above)
Comment (joeyaiello) [inline README.md]: With this change, I can approve

**Review by joeyaiello — CHANGES_REQUESTED** [review]

**Pull request merged**